modules/govuk/manifests/apps/ckan.pp

```diff
@@ -173,5 +184,17 @@
       owner  => 'deploy',
       group  => 'deploy',
     }
+
+    exec { 'symlink_pycsw_config':
+      command => "test -e ${ckan_home}/pycsw.cfg || ln -s ${pycsw_src}/default.cfg ${ckan_home}/pycsw.cfg",
+    }
+
+    exec { 'setup_pycsw_tables':
+      command => "test -e ${ckan_home}/pycsw_tables_created.tmp || ${ckan_bin}/paster --plugin=ckanext-spatial ckan-pycsw setup -p ${ckan_home}/pycsw.cfg && sudo touch ${ckan_home}/pycsw_tables_created.tmp",
+    }
+
+    exec { 'startup_pycsw_server':
+      command => "ps -C \"python */pycsw/csw.wsgi\" >/dev/null && echo \"pycsw is running\" || eval \". ${ckan_bin}/activate && python ${pycsw_src}/csw.wsgi &\"",
```

Comment:
Do you know if running using the builtin WSGI server supports handling requests concurrently?

Reply:
Not sure, I doubt there would be that many requests, the migration happened in January, we just recently received a ticket about the missing `csw` endpoint so I doubt that it would need to handle much traffic.

Reply:
Looks like using gunicorn is now the plan, and that should be fine.